src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportPowerShellDataFile.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Management.Automation;
+using System.Management.Automation.Language;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Import-PowerShellDataFile command.
+    /// </summary>
+    [Cmdlet(VerbsData.Import, "PowerShellDataFile", DefaultParameterSetName = "ByPath",
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=623621", RemotingCapability = RemotingCapability.None)]
+    public class ImportPowerShellDataFileCommand : PSCmdlet
+    {
+        bool _isLiteralPath;
+
+        /// <summary>
+        /// Path specified, using globbing to resolve
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0, ParameterSetName = "ByPath")]        
+        [ValidateNotNullOrEmpty]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public string[] Path { get; set; }
```

Comment:
We don't need a change as this code is just following a somewhat broken pattern.

If `LiteralPath` was set, `_isLiteralPath` will be set to true.  Then, if `Path` is set, `_isLiteralPath` is not cleared.  This isn't really a scenario I guess, but it is handled correctly in some places in our code base, but in many others it is not.